app/controllers/resources_controller.rb

```diff
@@ -1,8 +1,60 @@
 class ResourcesController < ApplicationController
-  before_action :logged_in_user, only: :index
+  before_action :logged_in_user, only: %i(create new index)
```

Comment:
```suggestion
  before_action :logged_in_user
```
ri luôn cho ngầu em... vào dc controller này phải đã login rồi nhé =))